### PR TITLE
Allow setting of phone number country for formatter

### PR DESCRIPTION
**Pull request by hiemanshu**

### CODE REVIEW

File: src/lib/index.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const getValidatedData = (data, optOuts) => {
   const missingCellRows = result[1]
 
   validatedData = _.map(validatedData, (row) => _.extend(row, {
-    cell: getFormattedPhoneNumber(row.cell) }))
+    cell: getFormattedPhoneNumber(row.cell, process.env.PHONE_NUMBER_COUNTRY || 'US') }))
```
Comment (shakalee14) [inline src/lib/index.js:41]: Great - we should update our .env template files too if we're adding another variable

**Comment by hiemanshu** [comment]
Done. Added the variable to env template.

**Review by sandramchung — APPROVED** [review]
LGTM

**Comment by sreynen** [comment]
Deployed and successfully tested on stage.

**Pull request merged**